src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -45,53 +54,23 @@ public ClimberSide(int motorId, boolean isMotorReversed) {
           motor.setSoftLimit(SoftLimitDirection.kReverse, 0),
           motor.enableSoftLimit(SoftLimitDirection.kReverse, true),
           motor.setSoftLimit(SoftLimitDirection.kForward, ClimberConstants.FORWARD_SOFT_LIMIT),
-          motor.enableSoftLimit(SoftLimitDirection.kForward, false));
+          motor.enableSoftLimit(SoftLimitDirection.kForward, false),
+          controller.setP(kP),
+          controller.setI(kI),
+          controller.setD(kD));
```

Comment:
are we using this at some point? if not, remove the field and constants. IJ should be able to recursively safe delete them once they're unused.